queue/src/workers/generate-mails.ts

```diff
@@ -1,4 +1,4 @@
-import { PutObjectCommand } from '@aws-sdk/client-s3';
+const { Upload } = require('@aws-sdk/lib-storage');
```

Comment:
upload est déprécié dans l'API AWS v2, c'est celle ci qui est recommandé pour l'API v3

Reply:
C'est expliqué ici :

https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/migrating/notable-changes/

_In v2, the S3 client contains an [upload() ](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property)
operation that supports uploading large objects with [multipart upload feature offered by S3 ](https://docs.aws.amazon.com/AmazonS3/latest/userguide/mpuoverview.html)_

_In v3, [@aws-sdk/lib-storage package ](https://github.com/aws/aws-sdk-js-v3/blob/main/lib/lib-storage)
is available. It supports all the features offered in v2 upload(), and supports both Node.js and browsers runtime._